components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -123,6 +123,9 @@ const hostDashboardMetaDataQuery = gql`
       @include(if: $getViewCounts) {
       totalCount
     }
+    error: expenses(host: { slug: $hostSlug }, limit: 0, offset: 0, status: ERROR) @include(if: $getViewCounts) {
```

Comment:
Woot? I didn't know our Apollo Server already supports `@include` directives; that's a game-changer! Would be worth sharing on slack, I don't think anyone in the engineering team is aware of that

Reply:
@Betree Had to double-check it actually works now, and thankfully yes, the server does not resolve those fields when the value is false

Reply:
Cool! Let's let the others know in `#engineering` :-)